client/influxdb.go

```diff
@@ -13,8 +13,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/influxdb/influxdb/influxql"
-	"github.com/influxdb/influxdb/tsdb"
+	"github.com/influxdb/influxdb/models"
```

Comment:
lol, it's the only name I could come up with.  I'm sure @benbjohnson will still hate it :smiling_imp: